plugins/item_tasks/web_client/stylesheets/selectTaskWidget.styl

```diff
@@ -0,0 +1,16 @@
+.g-task-custom-target-container
+  margin-bottom 8px
+
+  .g-target-result
+    padding 5px 10px
+    background-color #eee
+    border-radius 3px
+    margin-bottom 6px
+
+.g-selected-task
+  color #555
+  position relative
+  display block
+  padding 10px 15px
+  background-color #fff
```

Comment:
Do you need to specify the `background-color`? I think it's already white by default.